components/com_apps/models/category.php

```diff
@@ -256,6 +256,9 @@ private function loadExtensionIDs($db, $catid)
 	
 	public function getExtensions()
 	{
+		
+		return array();
```

Comment:
Due to return-line added at begin of function, is the now unreachable code below left on purpose or is this WIP ?